src/test/java/org/folio/rest/utils/TestEntities.java

```diff
@@ -2,9 +2,11 @@
 
 
 import org.folio.rest.jaxrs.model.Invoice;
+import org.folio.rest.jaxrs.model.InvoiceLine;
 
 public enum TestEntities {
-  INVOICE("/invoice-storage/invoices", Invoice.class, "invoice.sample", "note", "Updated note for invoice", 0);
+  INVOICE("/invoice-storage/invoices", Invoice.class, "invoice.sample", "note", "Updated note for invoice", 0),
+  INVOICE_LINES("/invoice-storage/invoice-lines", InvoiceLine.class, "invoice_line.sample", "quantity", "5", 3);
```

Comment:
`quantity` is of [integer type](https://github.com/folio-org/acq-models/blob/master/mod-invoice-storage/schemas/invoice_line.json#L69) so instead of `"5"`, it should be `5`?

Reply:
Yes, you are totally correct about Integer. But in the TestEntities enum updatedFieldValue declared as String. And, if I understand correctly, in the CrudTest this field can be operated as String.

Reply:
Good point. Then in that case, [this](https://github.com/folio-org/mod-invoice-storage/blob/master/src/test/java/org/folio/rest/impl/TestBase.java#L130)will always be compared against `String` datatype. Is this a bug then ...

Reply:
I think it is not a bug because here we check the operations at the DB layer, namely, whether we get the same object that we previously saved in the DB. But for generalization and greater clarity, we can present updatedFieldValue field as an Object type so as not to confuse further. I did this in the last commit.